openshift/templates/httpd.json

```diff
@@ -45,7 +45,10 @@
       "kind": "Route",
       "apiVersion": "v1",
       "metadata": {
-        "name": "${NAME}"
+        "name": "${NAME}",
+        "annotations": {
+          "template.openshift.io/expose-uri": "http://{.spec.host}{.spec.path}"
```

Comment:
Since I was following along, what's the significance of this? Is there some doc explaining what this annotation does? Sorry for the hijacking this thread.